File: Publisher Integration/SPX Mediation/Unified Bidding | iOS Integration/Location Update.m
/** CLLocationManagerDelegate method */
- (void)locationManager:(CLLocationManager *)manager didUpdateLocations:(NSArray<CLLocation *> *)locations {
    CLLocation *userLocation = locations.lastObject;
    if (userLocation) {
        SmaatoSDK.userLocation = [[SMALocation alloc] initWithLatitude:userLocation.coordinate.latitude
                                                             longitude:userLocation.coordinate.longitude
                                                    horizontalAccuracy:userLocation.horizontalAccuracy
                                                             timestamp:userLocation.timestamp];
    }
}

File: Publisher Integration/SPX Mediation/Unified Bidding | iOS Integration/Location Update.swift
/** CLLocationManagerDelegate method */
func locationManager(_ manager: CLLocationManager, didUpdateLocations locations: [CLLocation]) {
    guard let userLocation = locations.last else {
        return
    }
    SmaatoSDK.userLocation = SMALocation(latitude:userLocation.coordinate.latitude,
                                        longitude:userLocation.coordinate.longitude,
                               horizontalAccuracy:userLocation.horizontalAccuracy,
                                        timestamp:userLocation.timestamp)
}